### PR TITLE
Added some files

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+module.exports = {
```
Comment (adamweeks) [inline .eslintrc.js]: Could we use the same file format as the components and call this `.eslintrc`?

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  parser: "babel-eslint",
+  extends: ["plugin:prettier/recommended"],
```
Comment (adamweeks) [inline .eslintrc.js]: Let's use the webex eslint base config:
https://www.npmjs.com/package/@webex/eslint-config-base

Comment (akoushke) [reply .eslintrc.js]: then we need to add that to `webex-components` too.

Comment (adamweeks) [reply .eslintrc.js]: It already is. [eslint-config-react](https://github.com/webex/components/blob/master/.eslintrc#L4) includes the base config and adds react linting.

**Pull request merged**